totp/__init__.py

```diff
@@ -41,20 +42,27 @@ def get_pass_entry(path):
 
 
 def copy_to_clipboard(text):
-    selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
     try:
+        if platform.system() == 'Linux':
+            selection = os.environ.get('PASSWORD_STORE_X_SELECTION', 'clipboard')
+            command = ['xclip', '-selection', selection]
+        elif platform.system() == 'Darwin':
+            command = ['pbcopy']
+        else:
```

Comment:
I think that `xclip` should be used as the fallback, since it'll work on just about any OS that runs X (I think only windows doesn't).